File: Java/217.java
import java.util.Scanner;

/**
 *
 * 217 - ¿Qué Lado de la Calle? - Iván - https://github.com/wildfireOfMine
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int registro = sc.nextInt();
        while (registro != 0) {
            if (registro % 2 == 0) {
                System.out.println("DERECHA");
            } else {
                System.out.println("IZQUIERDA");
            }
            registro = sc.nextInt();
        }
    }

}

File: C/217.c
#include <stdio.h>
#include <stdlib.h>

// 217 - ¿Qué Lado de la Calle? - Iván - https://github.com/wildfireOfMine

int main()
{
    int number = 0;
    scanf("%d", &number);
    while (number != 0) {
       if (number % 2 == 0) {
            printf("DERECHA\n");
       } else {
            printf("IZQUIERDA\n");
       }

       scanf("%d", &number);
    }
    return 0;
}
